app/tr_php/01/task02.php

```diff
@@ -12,8 +12,12 @@
  * ・コネクト太郎さんに、「おはようございます！！」と挨拶をするメッセージを、$message_oneに代入してください。
  * ・コネクト次郎さんに、「おやすみなさい！！」と挨拶をするメッセージを、$message_twoに代入してください。
 */
-$message_one;
-$message_two;
+function getGreetingMessage($name, $greeting) {
+    return "$greeting, $name!";
+}
+
+$message_one = getGreetingMessage("Connect Taro", "Good morning!!");
+$message_two = getGreetingMessage("Connect Jiro", "Good night!!");
```

Comment:
固定の文字列（特殊文字や変数展開をしないもの）を定義する場合は、シングルクォーテーションが適しています。
シングルクォーテーションで囲むとき、ダブルクォーテーションで囲むときの違いを使い分けましょう。

```suggestion
$message_one = getGreetingMessage('Connect Taro', 'Good morning!!');
$message_two = getGreetingMessage('Connect Jiro', 'Good night!!');
```

Reply:
この辺りは横展開で修正お願いします。

Reply:
ご指摘、ありがとうございます！
改めてシングルクォーテーションとダブルクォーテーションで囲むときの違いについて復習しました。
横展開の関連ファイルの該当する箇所も修正しました。